ci/azure-pipelines.yml

```diff
@@ -14,6 +14,38 @@ variables:
   GOVER: 1.14.4
 
 stages:
+  - stage: DocBuild
+    dependsOn: []
+    jobs:
+    - job: DocBuild
+      pool:
+        vmImage: ubuntu-18.04
+      container:
+        image: btl5037/tox
+      steps:
+        - checkout: self
+          path: 'go/src/github.com/hyperledger/fabric'
+          displayName: Checkout Fabric Code
+        - script: tox -edocs
+          displayName: Build Documentation
+        - publish: 'docs/_build/html'
+          displayName: Publish Documentation Artifacts
+
+  - stage: LinkCheck
+    dependsOn: []
+    jobs:
+      - job: LinkCheck
+        pool:
+          vmImage: ubuntu-18.04
+        container:
+          image: btl5037/tox
```

Comment:
Where's the Dockerfile used to generate this image? I don't see it in the ci folder.

Reply:
I have it, I just haven't pushed it yet as I am trying to prune the dependencies out of it to make it as small as possible. I am only pushing it here to test. Should have opened as a draft.

Reply:
It's also going to artifactory at the end, I'm not going to host it

Reply:
FYI I ran a free link checker as a sanity check on the Fabric docs and showed ~40 out of 2k+ links broken.  I ran a cursory check of a few and it seemed pretty accurate.  I'll bring this up with the docs group and we'll start fixing them under a separate PR(s).  Thanks for doing this Brett.

Reply:
BTW, deadlinkchecker.com is the sanity check tool I ran.

Reply:
@denali49 Yea, I think there are only a handful that aren't real (mostly example URL's) which I need to figure out how to explicitly ignore a link in sphinx